### ISSUE TITLE
Are all properties now intended to be readonly since `v0.6.1`? Error "TypeError: Cannot assign to read only property 'OtherProperty' of object '#<Object>'" after the update

### ISSUE THREAD

**Issue by ceisele-r** [Leonidas-from-XIV/node-xml2js#689]
After updating from `v0.6.0` to `v0.6.1`, we are facing issues like the following:
```
TypeError: Cannot assign to read only property 'OtherProperty' of object '#<Object>'
```

The issue occurs when doing something like this:
```typescript
const parser = new Parser();
const data = await parser.parseStringPromise(fileXml);
data.SomeProperty.OtherProperty = ["some"]; // Error happens here
```

Is it intended that all properties are now readonly (/is this the result of the recent CVE fixes)?
I am wondering because when looking at the diff from `v0.6.0` to `v0.6.1`, [here](https://github.com/Leonidas-from-XIV/node-xml2js/blob/8e9a1209235406fe10cf96f4d458ce46c94bf412/lib/parser.js#L38) it seems that properties should still be defined writable.

**Comment by Leonidas-from-XIV** [comment]
It is definitely not meant to be like this (and that would be a breaking change), so I am indeed surprised that they are read-only. As you point out this is a fix for the recent CVE that does not rely on filtering names so I am rather surprised it breaks in such a way. Need to investigate what is going on.

Gladly accepting patches to fix this.

**Comment by Leonidas-from-XIV** [comment]
I figured it out, the `writable` attribute was mistyped. In my tests things would just silently fail instead of showing an error but they would fail indeed. I've released 0.6.2 fixing this.

**Comment by ceisele-r** [comment]
@Leonidas-from-XIV thanks for the super fast resolution!
This makes sense, it failed for us in strict mode.
